sigstore-java/src/main/java/dev/sigstore/KeylessSigner.java

```diff
@@ -152,10 +151,49 @@ public Builder sigstoreStagingDefaults()
     }
   }
 
-  public KeylessSigningResult sign(Path artifact)
+  /**
+   * Convenience method to convert files to their digests
+   *
+   * @param artifacts paths to files to convert
+   * @return an array of digests in the same order as the entered artifacts
+   * @throws IOException if issues occur processing the file contents
+   */
+  public static byte[][] asDigests(Path... artifacts) throws IOException {
```

Comment:
Do we have a class for `Hash`? (e.g. algorithm + bytes)
Could this be `List<Hash>`?

Reply:
I had the varargs earlier because I was using multiple methods with similar signatures -- where lists wouldn't work. I think this can all be lists though. Not quite sure about `Hash` yet, lemme see.

Reply:
I see why varargs might be helpful, so we might consider both `asDigests(Path... artifacts)` and `asDigests(Iterable<? extends Path> artifacts)`. However, my question here was regarding the method result type: use `List` instead of array.

Reply:
yeah it made it easier to just drop it into the varargs sign...